src/handle_action.cpp

```diff
@@ -142,6 +143,7 @@ input_context game::get_player_input( std::string &action )
     } ) {
```

Comment:
Should also indent this list to make it look better.

Reply:
Done,